docs/gen-1/red-blue/main-glitchless/beginner-route/lass/README.md

```diff
@@ -707,9 +767,9 @@ After Lance:
 - [OPTIONAL SAVE: Saving for Champ can be good if you don't have a spare X Accuracy + X Special + Revive]
 
 **Champion:**
-- Pidgeot: X Spec
-	- If Pidgeot does NOT use Sky Attack turn 1:
-		- Pidgeot: X Acc + HD
+Pidgeot: **Turn 1: USE X SPECIAL**, then adapt:
+	- If Pidgeot does NOT use Sky Attack on turn 1:
+		- Pidgeot: use X Acc on turn 2, HD
```

Comment:
You broke the formatting, on top of that i'm not sure adding that many words make it clearer

Reply:
Got it, reverting this back entirely to how it was.